beacon-chain/state/state-native/BUILD.bazel

```diff
@@ -38,13 +47,11 @@ go_library(
         "state_trie.go",
         "types.go",
         "validator_index_cache.go",
-    ] + select({
```

Comment:
qq why was this moved?

Reply:
Because it was causing gazelle to throw a warning / error